ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -867,6 +867,37 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-upi
+- as: azure-ipi-fullyprivate-proxy-f360-compliance-operator
```

Comment:
We encourage teams to use arm arch to run arch-independent testing to save cost, does the testing depend on amd arch ?

Reply:
compliance operator doesn't support arm yet.

Reply:
Got it.